portal/package.json

```diff
@@ -23,15 +23,15 @@
     "js-cookie": "2.2.1",
     "numeral": "2.0.6",
     "uuid-validate": "0.0.3",
-    "vue": "2.6.10",
+    "vue": "^2.6.11",
```

Comment:
Kun je de patch ook vastzetten door dakje te verwijderen. Ik vertrouw JS community niet voldoende om dit vrij te laten. Eens in de zoveel tijd` npm run outdated` en dependabot in de gaten houden werkt in mijn ervaring beter dan hopen dat builds blijven werken

Reply:
Dit is juist om dependabot te laten werken. Als we het vast zetten dan gaat dependabot ook niet updates door geven. Zolang we de lockfile niet aanpassen wijzigt er ook niks.

Reply:
We wijzigen altijd de lockfile, want we doen een clean build per deploy. Ik weet dat bij Python dependabot suggesties geeft ook als je versie pinned is. Ik weet vrij zeker dat het dit ook voor JS doet, want ik werk eigenlijk altijd met pinned versions. Is dat dan veranderd?